Resources/doc/configuration.md

```diff
@@ -178,3 +178,13 @@ idci_step:
                                         last_name: '{{ flow_data.data.personnal.last_name|raw }}'
 ```
 
+Use custom Twig extensions
+--------------------------
+
+If you want to use a extension written by you in your project that contains a map, you can add in your idci_step.yaml file in node `idci_step`:
```

Comment:
If you need to add twig extensions as part of the Step/Path Events provided by this bundle, you can declare them in the configuration file `config/packages/idci_step.yaml`